RELEASE-NOTES.md

```diff
@@ -64,7 +64,7 @@ André Bargull (60):
 *     Add missing @Override annotations
 *     Add missing generic type info to deprecatedsrc/
 *     Add missing generic type info to toolsrc/
-*     Add missing generic type info to testsrc/
+*     Add missing generic type info to src/test/resources/
```

Comment:
Another minor thing: I don't think a changelog should be updated -- even if folder is no longer there, it's certainly existed when it was written.